src/blue.ts

```diff
@@ -354,17 +359,17 @@ export function blueProxyFactory(env: MembraneBroker) {
     // future optimization: hoping that proxies with frozen handlers can be faster
     freeze(BlueDynamicProxyHandler.prototype);
 
-    function getBlueValue(red: RedValue): BlueValue {
+    function getBlueValue<T>(red: T): T {
```

Comment:
retypes of this very important method to use a generic type inference, whatever comes in, is what comes out.